src/app/login/AccountRecoveryRequestForm.tsx

```diff
@@ -106,26 +109,36 @@ export default function AccountRecoveryRequestForm({
         </DialogActions>
       </Dialog>
 
-      <Dialog open={openPrompt} data-testid="login-dialog-email-send-msg">
-        <DialogTitle>{title}</DialogTitle>
-        <DialogContent>
+      <Dialog 
+        open={openPrompt} 
+        data-testid="login-dialog-email-send-msg"
+        fullWidth={true}
+        maxWidth='sm'
+        style={{  padding: '100px', borderRadius: '20px 20px 0 0' }}>
+        <DialogTitle style={{ textAlign: 'center', fontSize: '24px', marginTop: '10px'  }}>{title}</DialogTitle>
+        <DialogContent style={{ padding: '20px', margin: '10px', fontSize: '20px'}}>
           <p>{message}</p>
         </DialogContent>
-        <DialogActions>
-          <Link href={'/login/recover-account'}>Continue</Link>
+        <DialogActions style={{ justifyContent: 'center', margin: '10px' }}>
+          <Button>
+            <Link style={{ textAlign: 'center' }} href={'/login/recover-account'}>Continue</Link>
+          </Button>
         </DialogActions>
       </Dialog>
 
       <Dialog
         open={openPopup}
         onClose={handleClosePopup}
         data-testid="recover-dialog-error-msg"
+        fullWidth={true}
+        maxWidth='sm'
+        style={{  padding: '100px', borderRadius: '20px 20px 0 0' }}
```

Comment:
referring to the comment above; you can define props if you want to add more custom styling